awx/main/tests/functional/models/test_notifications.py

```diff
@@ -117,7 +120,8 @@ def test_context(self, JobClass, sqlite_copy_expert, project, inventory_source):
     @pytest.mark.django_db
     def test_schedule_context(self, job_template, admin_user):
         schedule = Schedule.objects.create(name='job-schedule', rrule='DTSTART:20171129T155939z\nFREQ=MONTHLY', unified_job_template=job_template)
-        job = Job.objects.create(name='fake-job', launch_type='workflow', schedule=schedule, job_template=job_template)
+        kwargs = {}
```

Comment:
I'm not sure why kwargs is set back to empty dict here

Reply:
neither am I, deleting